src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/SpatialCoordinateSystemManager.cs

```diff
@@ -523,6 +523,15 @@ private void RegisterEvents(INetworkManager networkManager)
             networkManager.RegisterCommandHandler(CoordinateStateMessageHeader, OnCoordinateStateReceived);
             networkManager.RegisterCommandHandler(SupportedLocalizersMessageHeader, OnSupportedLocalizersMessageReceived);
             networkManager.RegisterCommandHandler(SpatialCoordinateSystemParticipant.LocalizationDataExchangeCommand, OnParticipantDataReceived);
+            
+            if (networkManager.IsConnected)
+            {
+                var connections = networkManager.Connections;
+                foreach (var connection in connections)
+                {
+                    OnConnected(connection);
+                }

```

Comment:
If OnConnected caused the connection status to change (e.g. a disconnect), this could potentially cause enumeration over a modified collection. Might be safer to create a snapshot copy of the connections to visit.